openedx/core/djangoapps/theming/helpers.py

```diff
@@ -52,3 +72,145 @@ def get_themed_template_path(relative_path, default_path, **kwargs):
     if is_stanford_theming_enabled and not is_microsite:
         return relative_path
     return microsite.get_template_path(default_path, **kwargs)
+
+
+def get_template_path_with_theme(relative_path):
+    """
+    Returns template path in current site's theme if it finds one there otherwise returns same path.
```

Comment:
All the docstrings in this file should be written using the Google style:

https://github.com/edx/edx-platform/wiki/Python-Guidelines

```
def func(arg1, arg2):
    """
    Summary line.

    Extended description of function.

    Arguments:
        arg1 (int): Description of arg1
        arg2 (str): Description of arg2

    Returns:
        bool: Description of return value

    """
```